### ISSUE TITLE
Sympa::List::_cache_get always return undef

### ISSUE THREAD

**Issue by cozy517** [sympa-community/sympa#1182]
Version
-------
6.2.60

Installation method
-------------------
yum install from epel

Expected behavior
-----------------
call $self->_cache_get('edit_list_conf') in Sympa/List.pm, get cached HASH reference when edit_list.conf is not updated.
I mean edit_list.conf's mtime is not change.

Actual behavior
---------------
Every time returned undef from _cache_get function. Then repeat open file unexpectedly often.

Additional information
----------------------
File's mtime when last time read.
```
    my $lasttime = $self->{_mtime}{$type};
```
File's mtime just now.
```
        $mtime = $self->_cache_read_expiry($type);
``
If the file is not updated since last opened(usual case) $lasttime and $mtime are same, then returned undef instead of cached data.
```
    return undef if $lasttime <= $mtime;
```

**Comment by ikedas** [comment]
Hi @cozy517 ,
The issue may be a duplicate of #1090 fixed in Sympa 6.2.62.  Could you please check the recent version?

**Comment by cozy517** [comment]
Thank you for inform and rightly guessed what I mean.
Yes, my actual problem was reading edit_list.conf takes too long.

I just checked _cache_get function is not fixed yet on version 6.2.63b before post.
Now I reallized totally fixed this problem by updating _load_edit_list_conf, may_edit  etc.
And I understand current cache mechanism will be gone.
